### PR TITLE
Clean up Create Account Button A/B test

### PR DESCRIPTION
**Pull request by kelbyhawn**
Cleans up and integrates the Create account button in the header navigation and hamburger now that the experiment is over and the Create account button test bucket won 🎉 

## Links
Jira ticket: [ACQ-1936](https://codedotorg.atlassian.net/browse/ACQ-1936)
Tech spec: [here](https://docs.google.com/document/d/1RDQv2IAjkdJvxfxWFgisRNM9GuJInfqwf7v4Muifgt8/edit#heading=h.4schd43y1qoc)
Statsig experiment: [here](https://console.statsig.com/6uqJc02CIZP7hHowwYO7AS/experiments/create_account_button_2/results)

## Testing story
Tested locally and added unit and UI tests

----

## code.org

https://github.com/user-attachments/assets/3927c073-8e78-4e1c-bc02-8dee4d901f90

## studio.code.org

https://github.com/user-attachments/assets/83b7d6ef-2003-47ee-a4ad-291c2ddbb6a1

## Activity level

https://github.com/user-attachments/assets/f8237e4c-9c80-4424-8bc8-55530c372287

## Updated `signin_callout_wrapper` popover

| Before | After |
| ---- | ---- |
| Before | After |

### CODE REVIEW

File: apps/src/code-studio/components/header/SignInCallout.jsx [hunk 1]
```diff
@@ -110,8 +110,11 @@ const styles = {
     width: 400,
     textAlign: 'left',
     whiteSpace: 'normal',
+    color: 'white',
+    fontWeight: '400',
```
Comment (kelbyhawn) [inline apps/src/code-studio/components/header/SignInCallout.jsx:114]: I'm not using variables here since they aren't being used in this file and refactoring this further feels out of scope.

File: shared/haml/hamburger.haml [hunk 2]
```diff
@@ -1,6 +1,5 @@
 :ruby
   options = { level: level, script_level: script_level, language: language, user_type: user_type, loc_prefix: loc_prefix, request: request }
-  user_header_options = { loc_prefix: loc_prefix }
```
Comment (kelbyhawn) [inline shared/haml/hamburger.haml:3]: Cleaned this up to reduce confusion.

File: dashboard/test/ui/features/foundations/hamburger.feature [hunk 3]
```diff
@@ -49,6 +51,27 @@ Feature: Hamburger dropdown
     And I see "#support"
     And I see "#report-bug"
 
+  @only_phone
```
Comment (kelbyhawn) [inline dashboard/test/ui/features/foundations/hamburger.feature:54]: I wasn't able to test these locally, but they are passing drone:

Screenshot 2024-08-15 at 7 56 35 AM

**Review by hannahbergam — APPROVED** [review]
Looks great! We reviewed live and went over all translation and site cases as well

File: dashboard/test/ui/features/foundations/user_menu.feature [hunk 4]
```diff
@@ -7,6 +7,18 @@ Scenario: Signed Out - sign in button shows
   And I wait until element "#signin_button" is visible
   And I wait until element ".display_name" is not visible
 
+Scenario: Signed Out - create account button shows on code.org page
```
Comment (bethanyaconnor) [inline dashboard/test/ui/features/foundations/user_menu.feature:10]: Should these also be `no mobile`?

File: shared/haml/hamburger.haml (hunk 2 above)
Comment (bethanyaconnor) [reply shared/haml/hamburger.haml:3]: Thank you!

**Review by bethanyaconnor — APPROVED** [review]
One question but looks great!

File: dashboard/test/ui/features/foundations/user_menu.feature (hunk 4 above)
Comment (kelbyhawn) [reply dashboard/test/ui/features/foundations/user_menu.feature:10]: `@no_mobile` is set at the very top on line 1 for all the tests.
